toptal/gitignore.io
# updated templates from https://github.com/toptal/gitignore

# Description
This PR synchronizes templates from https://github.com/toptal/gitignore

https://github.com/toptal/gitignore/compare/a3f813d...d6f8a64

# How to test

- `docker-compose up --build`
- go to http://localhost:8080/api/video
- verify `*.mkv` included in the response
